Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -1701,6 +1701,8 @@ def lhegensim(fragment,howMuch):
 defaultDataSets={}
 defaultDataSets['2017']='CMSSW_8_1_0_pre16-81X_upgrade2017_realistic_v22-v'
 defaultDataSets['2017Design']='CMSSW_8_1_0_pre16-81X_upgrade2017_design_IdealBS_v6-v'
+defaultDataSets['2018']='CMSSW_8_1_0_pre16-81X_upgrade2017_realistic_v22-v'
+defaultDataSets['2018Design']='CMSSW_8_1_0_pre16-81X_upgrade2017_design_IdealBS_v6-v'
```

Comment:
Yes, they'll be appropriate for the new 2018, but not the new 2017. We don't have any alternatives at the moment, though. It should be updated once we have new relvals.